bin/node/runtime/src/lib.rs

```diff
@@ -777,7 +779,7 @@ impl pallet_nomination_pools::Config for Runtime {
 	type RewardCounter = FixedU128;
 	type BalanceToU256 = BalanceToU256;
 	type U256ToBalance = U256ToBalance;
-	type StakingInterface = pallet_staking::Pallet<Self>;
+	type Staking = pallet_staking::Pallet<Self>;
```

Comment:
```suggestion
	type Staking = Staking; 
```

Reply:
I was being consistent with the other Staking declarations in this node config. Should we then revisit those too?

Reply:
No strong opinion, I think historically we use more the type alias rather than `Pallet` directly. 

Here, I suggested it for aesthetics. `X = X` looks better :D 

Personally, I dislike magic keyword that are not defined (`type Staking` is not visible in this scope), so I actually like yours better. 

Beyond the scope of this PR, these are all my thoughts on the matter, do as you prefer.

Reply:
fixed